Tasks/AzureRmWebAppDeployment/LegacyUtils/Utility-Legacy.ps1

```diff
@@ -153,17 +153,19 @@ function Run-Command
 	{
         if( $psversiontable.PSVersion.Major -le 4)
         {
-           cmd.exe /c "`"$command`""
+           cmd.exe /c "`"$command`"" 2>&1
         }
         else
         {
-           cmd.exe /c "$command"
+           cmd.exe /c "$command" 2>&1
```

Comment:
Will this resolve the bug I was talking about yesterday? when the task was showing green even though it failed internally